src/ridgeplot/_color/interpolation.py

```diff
@@ -134,6 +138,8 @@ def _interpolate_trace_index(ctx: InterpolationContext) -> ColorscaleInterpolant
 
 
 def _interpolate_trace_index_row_wise(ctx: InterpolationContext) -> ColorscaleInterpolants:
+    if ctx.n_rows <= 1:
+        return [[0.0] * len(row) for row in ctx.densities]
```

Comment:
I think here you're missing the case where the number of rows might be >1 but one of the rows only contains a single trace. Let me know if you need help with this 👍

Reply:
help me with it, please 👍

Reply:
I'll push some test cases that should fail under this implementation. Let me know if they are helpful 👍

Reply:
hellooo. Sorry for my late response, the last week was a busy week for me. I will start to work on it. Thank so much :)

Reply:
@tpvasconcelos heellooo. I tried some solutions, but the tests still fails :/ . Sorry for that

Reply:
No worries @sstephanyy! See https://github.com/tpvasconcelos/ridgeplot/pull/268/commits/31876a05a75d58cc68009e6403d9f20d15403d52 (this should fix it!)